core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -112,11 +115,15 @@ export class DotEditContentFormComponent implements OnInit {
     saveContenlet() {
         const formValue = this.form.getRawValue();
         this.formData.fields.forEach((field) => {
-            if (!FLATTENED_FIELD_TYPES.includes(field.fieldType as FIELD_TYPES)) {
-                return;
-            }
+            // Shorthand for conditional assignment
+
+            FLATTENED_FIELD_TYPES.includes(field.fieldType as FIELD_TYPES) &&
+                (formValue[field.variable] = formValue[field.variable]?.join(','));
 
-            formValue[field.variable] = formValue[field.variable]?.join(',');
+            CALENDAR_FIELD_TYPES.includes(field.fieldType as FIELD_TYPES) &&
+                (formValue[field.variable] = formValue[field.variable]
+                    ?.toISOString()
+                    .replace(/T|\.\d{3}Z/g, (match: string) => (match === 'T' ? ' ' : ''))); // To remove the T and .000Z from the date)
```

Comment:
Can't we just do this? https://primeng.org/calendar#api.calendar.interfaces.LocaleSettings.dateFormat

Reply:
No, that just modifies how it looks in the component, the model format is still `ISOString`

Reply:
https://primeng.org/calendar#format it works like a mask